package.json

```diff
@@ -214,10 +214,10 @@
     "stylelint-config-standard": "18.2.0",
     "stylelint-order": "2.0.0",
     "url-loader": "1.1.2",
-    "webpack": "4.27.1",
-    "webpack-cli": "3.1.2",
+    "webpack": "4.28.3",
+    "webpack-cli": "3.2.1",
     "webpack-dev-server": "3.1.14",
```

Comment:
when did this get updated?

Reply:
Yesterday 😜

Reply:
how did the 🤖 get passed my percy block 🤔

Reply:
I merged that with my superpowers 💪